src/zocalo/cli/configure_rabbitmq.py

```diff
@@ -155,20 +165,21 @@ def get_queue_specs(group: Dict) -> List[QueueSpec]:
     ]
 
     # Add dead-letter queues within the default exchange with the "dlq." prefix
-    qspecs.extend(
-        [
-            QueueSpec(
-                name=f"dlq.{name}",
-                vhost=vhost,
-                arguments={
-                    "x-queue-type": qtype,
-                },
-                auto_delete=False,
-                durable=True,
-            )
-            for name in group["names"]
-        ]
-    )
+    if dlq_pattern:
+        qspecs.extend(
+            [
+                QueueSpec(
+                    name=f"dlq.{name}",
```

Comment:
Does this assume that the `dead-letter-routing-key` will always be this form?

Reply:
Thanks for catching this oversight 👍